graphs/energy/nodes/households/households_solar_pvt_solar_radiation.ad

```diff
@@ -1,36 +1,35 @@
-# Dummy data is used for this technology, will be updated.
 - use = energetic
-- energy_balance_group = local power generation
-- output.electricity = 0.17
-- output.solar_thermal = 0.10
-- output.loss = elastic
 - groups = [cost_electricity_production, electricity_production, primary_energy_demand, sustainable_production, volatile_production, lv_net_supply, wacc_households]
 - presentation_group = solar_panel
+- graph_methods = [full_load_hours]
+- energy_balance_group = local power generation
 - merit_order.group = dynamic: solar_pv
 - merit_order.level = lv
 - merit_order.subtype = volatile
 - merit_order.type = producer
-- graph_methods = [full_load_hours]
-- availability = 0.98
-- free_co2_factor = 0.0
-- forecasting_error = 0.05
-- land_use_per_unit = 1.6000000000000001e-06
-- takes_part_in_ets = 0.0
-- part_load_efficiency_penalty = 0.1
-- part_load_operating_point = 0.7
-- electricity_output_capacity = 0.000272
-- heat_output_capacity = 0.0001
-- initial_investment = 217.328
+- output.electricity = 12.1
+- output.solar_thermal = 36.2
+- output.loss = elastic
+- availability = 0.999
 - ccs_investment = 0.0
-- cost_of_installing = 136.272
+- construction_time = 0.0
+- cost_of_installing = 172.0
 - decommissioning_costs = 0.0
-- fixed_operation_and_maintenance_costs_per_year = 4.62
-- variable_operation_and_maintenance_costs_per_full_load_hour = 0.0
-- variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour = 0.0
-- construction_time = 0.3
+- fixed_operation_and_maintenance_costs_per_year = 0.0
+- initial_investment = 326.0
 - technical_lifetime = 25.0
+- variable_operation_and_maintenance_costs_for_ccs_per_full_load_hour = 0.0
+- variable_operation_and_maintenance_costs_per_full_load_hour = 0.0
 - wacc = 0.02
+- electricity_output_capacity = 0.000894
+- forecasting_error = 0.05
+- free_co2_factor = 0.0
+- heat_output_capacity = 0.0003333333333333333
```

Comment:
Ik ben benieuwd waar deze waarde vandaan komt. In de NSA zie ik deze formule: =1/3/1000

Reply:
In de bron data staat dat er per 3m2 PVT panelen 1 kW warmteopwek is. Dit heb ik omgerekend naar 1/3 kW per 1m2 PVT panelen en daarna gedeeld door 1000 zodat het in MW wordt weergegeven.